include/faabric/scheduler/MpiWorld.h

```diff
@@ -40,6 +40,8 @@ class MpiWorld
 
     int getSize();
 
+    void getSizePtr(int* size);
```

Comment:
These methods are not _necessary_ per se. However, to be able to use the function pointer trick (and save lines of code) I must ignore the return value (i.e. void preferably). Additionally, function overloading does not work well with declaring a pointer to the function, hence the easy workaround was to define two methods with the properties I needed.

The immediate workaround if you'd rather see these methods gone would be not using the `fptr` trick and perform the regular calls to `MpiWorld`.

Reply:
See comment below. I'm not sure the `fptr` trick adds enough value to justify changing the main codebase.